src/pages/salesConfirmation.tsx

```diff
@@ -0,0 +1,165 @@
+import React, { ReactElement } from 'react';
+import { isArray, upperFirst } from 'lodash';
+import moment from 'moment';
+import { SALES_OFFER_PACKAGES_ATTRIBUTE, PRODUCT_DATE_ATTRIBUTE } from '../constants';
+import {
+    CustomAppProps,
+    NextPageContextWithSession,
+    SalesOfferPackage,
+    ProductWithSalesOfferPackages,
+    ProductDate,
+} from '../interfaces';
+import TwoThirdsLayout from '../layout/Layout';
+import CsrfForm from '../components/CsrfForm';
+import ConfirmationTable, { ConfirmationElement } from '../components/ConfirmationTable';
+import { getSessionAttribute } from '../utils/sessions';
+import { isProductWithSalesOfferPackages, isProductDateAttributeWithErrors } from '../interfaces/typeGuards';
+
+const title = 'Sales Confirmation - Fares Data Build Tool';
+const description = 'Sales Confirmation page of the Fares Data Build Tool';
+
+type SalesConfirmationProps = {
+    salesOfferPackages: SalesOfferPackage[] | ProductWithSalesOfferPackages[];
+    ticketDating: TicketDating;
+};
+
+interface TicketDating {
+    productDates: ProductDate;
+    startDefault: boolean;
+    endDefault: boolean;
+}
+
+export const buildSalesConfirmationElements = (
+    salesOfferPackages: SalesOfferPackage[] | ProductWithSalesOfferPackages[],
+    ticketDating: TicketDating,
+): ConfirmationElement[] => {
+    const confirmationElements: ConfirmationElement[] = [];
+    if (isProductWithSalesOfferPackages(salesOfferPackages)) {
+        salesOfferPackages.forEach(product => {
+            confirmationElements.push({
+                name: 'Product',
+                content: upperFirst(product.productName),
+                href: 'selectSalesOfferPackages',
+            });
+            product.salesOfferPackages.forEach(sop => {
+                confirmationElements.push({
+                    name: `${upperFirst(product.productName)} - Sales Offer Package`,
+                    content: sop.name,
+                    href: 'selectSalesOfferPackages',
+                });
+            });
+        });
+    } else {
+        salesOfferPackages.forEach(sop => {
+            confirmationElements.push({
+                name: 'Sales Offer Package',
+                content: sop.name,
+                href: 'selectSalesOfferPackages',
+            });
+        });
+    }
+    const startDate = moment(ticketDating.productDates.startDate)
+        .utc()
```

Comment:
This causes a problem with utc as it will go one day before. So on the productDate i entered 11 04 2020 but when you get to this page it will be the day before due to utc.